examples/static/source/Root.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Link } from 'react-router'
 
 const addBase = (path) => (
-  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path}`
+  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path.slice(1)}`
```

Comment:
Ah, fair enough, I saw `process.env` and figured we were not on the client. I guess there's a polyfill?